src/neuroconv/datainterfaces/ecephys/intan/intandatainterface.py

```diff
@@ -95,6 +68,12 @@ def __init__(
         es_key : str, default: "ElectricalSeries"
         """
 
+        if stream_id is not None:
+            warnings.warn(
+                "Use of the 'stream_id' parameter is deprecated and it will be removed after September 2024.",
+                DeprecationWarning,
+            )
+
         self.stream_id = stream_id
```

Comment:
Can remove this line then, yes?

Reply:
Or this is a soft cycle so you still allow setting it with the warning that it's undesired behavior

Reply:
The first iteration was wrong though. Thanks for pointing it out. Should be ammended now.